### PR TITLE
Jan 2019 update

### LINKED ISSUE #12: Rebranding to Azure DevOps
Should we consider rebranding the workshop title and its relevant document to Azure DevOps instead of VSTS?

### LINKED ISSUE #16: Need to update Visual Studio Team Services part in lab guide.
1. Now VSTS is Azure DevOps. Please update the lab guide accordingly from [Exercise 2.1](https://github.com/Microsoft/MCW-Continuous-delivery-in-VSTS-and-Azure/blob/master/Hands-on%20lab/HOL%20step-by-step%20-%20Continuous%20delivery%20in%20VSTS%20and%20Azure.md#task-1-create-visual-studio-team-services-account). 
1. Whenever there is new signup for Azure DevOps(VSTS). New interface comes up for that signup. 
1. Please let me know in case of any question.
snag_1ea7a08d

### LINKED ISSUE #17: December Update - Author Suggestions
Use the same scenario but to use Azure DevOps to solve the customer problem.

### PR DESCRIPTION
**Pull request by sdavis3**
resolves #12 
resolves #16 
resolves #17

### CODE REVIEW

**Comment by mwasham** [comment]
Nice! 
A few tweaks: 

- Rename the lab guide to HOL step-by-step - Continuous Delivery in Azure DevOps.md
- Create a folder called archive in the hands-on lab folder and move the unguided lab guide to it.
- Rename the WDS guide filenames using the new name as well.
- The WDS PPTX is missing alt-text in the graphics
- The common scenarios graphic in the WDS Trainer/Learner guide is the old VSTS image
- Copyright dates in the MD should be updated to 2019

**Comment by DawnmarieDesJardins** [comment]
@sdavis3 
Hi Shane,
I see you've removed the VSTS from the title, but you left the "and Azure" making the new title **Continuous Delivery in Azure DevOps and Azure**.  Please make a quick run through to make sure the title is correct in all the documents before we send this for SME review.  Walter will be changing the repo title so links will change to match the new title.
Thanks!
Dawnmarie

**Comment by sdavis3** [comment]
@DawnmarieDesJardins Thank you for catching those. I updated the pull request in addition to renaming the paths where VSTS was included. Also archived the unguided lab folder of images.

File: Hands-on lab/archive/HOL unguided - Continuous delivery in Azure DevOps.md [hunk 1]
```diff
@@ -120,81 +132,79 @@ Since this solution is based on Azure Platform-as-a-Service (PaaS) technology, i
 
 -   You have deployed the completed template to Azure that creates the three environments: dev, test, and production.
 
-## Exercise 2: Create a Visual Studio Team Services team project and Git Repository
+## Exercise 2: Create an Azure DevOps team project and Git Repository
 
 Duration: 15 Minutes
 
-Tailspin Toys has asked you to create a continuous delivery process for their development team. To do that without disrupting their current production application, you are tasked with creating an environment in Azure where this new process and workflow can be proven. Your first task is to create a Visual Studio Team Services Team Project with Git source control where their source code can be maintained.
+Tailspin Toys has asked you to create a continuous delivery process for their development team. To do that without disrupting their current production application, you are tasked with creating an environment in Azure where this new process and workflow can be proven. Your first task is to create an Azure DevOps Team Project with Git source control where their source code can be maintained.
```
Comment (dmckinstry) [inline Hands-on lab/archive/HOL unguided - Continuous delivery in Azure DevOps.md]: Not critical at this point, but going forward the term "Team Project" is renamed to "Project."
https://docs.microsoft.com/en-us/azure/devops/organizations/projects/about-projects?view=vsts&tabs=new-nav

**Comment by dmckinstry** [comment]
Overall I'm happy with it.  Good job finding all the term renames... There is one that slipped by in some places:  "Team Project" ==> "Project"

**Comment by sdavis3** [comment]
I just pushed an update to the couple remaining references of "Team Project" to "Project".

**Comment by DawnmarieDesJardins** [comment]
@sdavis3 @mwasham @Jeff-Opsgility 
Hello gentlemen, We have the approval needed to finalize this update. Please incorporate the feedback in the open issues, and close them out as appropriate. Once that's complete and documents have been edited, please let me know it's ready for QC/merge. Thanks, Dawnmarie

**Comment by sdavis3** [comment]
To the best of my knowledge, the updates have been incorporated and the issues will automatically close when the code is merged.

**Pull request merged**